src/Sulu/Bundle/AdminBundle/Resources/js/components/CircleSelection/withPercentageValues.js

```diff
@@ -0,0 +1,105 @@
+// @flow
+import React from 'react';
+import type {ComponentType} from 'react';
+import {buildHocDisplayName} from '../../utils/react';
+import type {SelectionData} from './types';
+
+type Props = {
+    containerHeight: number,
+    containerWidth: number,
+    maxRadius: number | typeof undefined,
+    minRadius: number | typeof undefined,
+    onChange: (value: ?SelectionData) => void,
+    usePercentageValues: boolean,
+    value: SelectionData | typeof undefined,
+};
+
+export default function withPercentageValues(Component: ComponentType<*>) {
+    class WithPercentageValuesComponent extends React.Component<Props> {
+        wrappedComponent = Component;
+
+        static defaultProps = {
+            maxRadius: undefined,
+            minRadius: undefined,
+            usePercentageValues: false,
+        };
+
+        handleChange = (value: ?SelectionData) => {
+            const {containerHeight, containerWidth, onChange} = this.props;
+
+            if (!value) {
+                onChange(value);
+
+                return;
+            }
+
+            onChange({
+                ...value,
+                left: value.left / containerWidth,
+                top: value.top / containerHeight,
+                radius: value.radius / containerWidth,
+            });
+        };
+
+        getTransformedMaxRadius = () => {
```

Comment:
Could be a computed getter for better perfomance.

Reply:
fixed